src/model/adapter.py

```diff
@@ -739,7 +738,7 @@ def minimize_distance(self):
         try:
             # Include growth rate measurement
             growth_rate = next(m for m in self.measurements if m['type'] == 'growth-rate')
-            index.append(constants.MODEL_GROWTH_RATE[self.model.id])
+            index.append(storage.get(self.model.id).growth_rate_reaction)
```

Comment:
You're using information from the meta model in a few places. Would it make sense to assign the result of `storage.get(self.model.id)` to an instance attribute?